test/test_collection.py

```diff
@@ -116,13 +132,13 @@ def test_delete_many(self):
 
     def test_watch(self):
         res = self.explain.watch()
-        self.assertIn("queryPlanner", res["stages"][0]["$cursor"])
+        self.assertTrue(self._recursiveIn("queryPlanner", res))
         self.collection.watch(pipeline=[{"$project": {"tags": 1}}],
                               batch_size=10, full_document="updateLookup")
         last_logger_payload = self.logger.cmd_payload
         res = self.explain.watch(pipeline=[{"$project": {"tags": 1}}],
                                  batch_size=10, full_document="updateLookup")
-        self.assertIn("queryPlanner", res["stages"][0]["$cursor"])
+        self.assertTrue(self._recursiveIn("queryPlanner", res))
```

Comment:
I actually prefer the previous behavior here because `res["stages"][0]["$cursor"]` helps us understand the structure of the explain response. `_recursiveIn` makes the command response more of a black box. Can you revert these changes?

Reply:
Done.